lib/strategy/unique_count.rb

```diff
@@ -0,0 +1,7 @@
+module Strategy
+  class UniqueCount
+    def self.execute(array:)
+      array.uniq.count
+    end
+  end
+end
```

Comment:
Easy strategies are fine implicitly tested by use cases using them. For anything more complex we could write throwaway TDD tests that would guide us through the process and then we could decide do we want to discard them or not.